lighthouse-core/test/audits/seo/font-size-test.js

```diff
@@ -234,4 +237,54 @@ describe('SEO: Font size audit', () => {
     expect(auditResult.score).toBe(1);
     expect(auditResult.notApplicable).toBe(true);
   });
+
+  describe('attributes source location', () => {
+    async function runFontSizeAuditWithSingleFailingStyle(style, nodeProperties) {
+      const artifacts = {
+        URL: {finalUrl: 'http://www.example.com'},
+        MetaElements: makeMetaElements(validViewport),
+        FontSize: {
+          analyzedFailingNodesData: [
+            {textLength: 1, fontSize: 1, node: {nodeId: 1, ...nodeProperties}, cssRule: style},
+          ],
+        },
+        TestedAsMobileDevice: true,
+      };
+      const auditResult = await FontSizeAudit.audit(artifacts, getFakeContext());
+      expect(auditResult.details.items).toHaveLength(1);
+      return auditResult;
+    }
+
+    it('to inline node stylesheet', async () => {
+      const auditResult = await runFontSizeAuditWithSingleFailingStyle({
+        type: 'Inline',
+      }, {
+        parentNode: {attributes: ['id', 'my-parent']},
+        localName: 'p',
+        attributes: ['class', 'my-p'],
+      });
+
```

Comment:
Oh my gosh wait yeah this file is completely different!

I'm sorry @connorjclark I got confused between two different font-size-test changes and approved this immediately after looking at the other one, too many reviews today my b 😢

Reply:
is there a separate review ongoing with `runFontSizeAuditWithSingleFailingStyle`...

Reply:
you even tried to warn me over chat and it didn't work! 😆

Reply:
nope, this was it.

Reply:
> nope, this was it.

there was one in #9354, though? https://github.com/GoogleChrome/lighthouse/pull/9400#discussion_r305069915

Reply:
yes... and I moved it here and asked patrick to review over chat :) a flawless system as you see